Makefile

```diff
@@ -22,9 +22,9 @@ CONTROLLER_GEN_BIN_PATH := $(shell which controller-gen)
 
 # Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
 ifeq (,$(shell go env GOBIN))
-GOBIN=$(shell go env GOPATH)/bin
+	GOBIN=$(shell go env GOPATH)/bin
```

Comment:
Idiomatic Makefiles define variables without indentation to denote that they are always global, so can you revert these changes?

Reply:
It was to differentiate easier which one is saved under which conditions, but I'll revert it.